exportReport/Utility/SpringUtilities.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 1995, 2008, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Caught me off guard I guess, was concerned by the corporate copyright...probably a non-issue.

Question for @psiinon and @thc202, how do we document licenses for extensions/addons? Do we need to? Do addon authors need to? Does this require some similar documentation to [pending PR 1697](https://github.com/zaproxy/zaproxy/pull/1697)?